From f697623efc8f8f050402c8009e7ac4c2f72a7d0d Mon Sep 17 00:00:00 2001
From: Adrian Reber <areber@redhat.com>
Date: Sun, 16 Jan 2022 11:17:33 +0000
Subject: [PATCH 103/249] ci: added .lgtm.yml file

A couple of months (or years) ago I looked into lgtm.com for CRIU. Today
on a pull request I saw result from lgtm.com for the first time and it
failed. Not sure what triggered the lgtm.com message into the CRIU
repository, but with the .lgtm.yml file in this commit lgtm.com can
actually build CRIU.

Signed-off-by: Adrian Reber <areber@redhat.com>
---
 .lgtm.yml | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)
 create mode 100644 .lgtm.yml

diff --git a/.lgtm.yml b/.lgtm.yml
new file mode 100644
index 000000000..a28c35de0
--- /dev/null
+++ b/.lgtm.yml
@@ -0,0 +1,30 @@
+extraction:
+  cpp:
+    prepare:
+      packages:
+      - "protobuf-c-compiler"
+      - "libprotobuf-c-dev"
+      - "libprotobuf-dev"
+      - "build-essential"
+      - "libprotobuf-dev"
+      - "libprotobuf-c-dev"
+      - "protobuf-c-compiler"
+      - "protobuf-compiler"
+      - "python3-protobuf"
+      - "libnet-dev"
+      - "pkg-config"
+      - "libnl-3-dev"
+      - "libbsd0"
+      - "libbsd-dev"
+      - "iproute2"
+      - "libcap-dev"
+      - "libaio-dev"
+      - "python3-yaml"
+      - "libnl-route-3-dev"
+      - "python-future"
+      - "gnutls-dev"
+    configure:
+      command:
+      - "ls -laR images/google"
+      - "ln -s /usr/include/google/protobuf/descriptor.proto images/google/protobuf/descriptor.proto"
+      - "ls -laR images/google"
-- 
2.35.1

